core/bug_api.php

```diff
@@ -847,6 +853,24 @@ function bug_cache_row( $p_bug_id, $p_trigger_errors = true ) {
 		}
 	}
 
+	if ( is_null( $t_row->token ) ) {
```

Comment:
@vboctor 

Token can be requested by other means, not just via UI. For example, SOAP API. Or any future code which might refer this feature.

I thought it would be slow to upgrade large DB.